pkg/crc/machine/config/config.go

```diff
@@ -10,7 +10,7 @@ type MachineConfig struct {
 	Name              string
 	Memory            uint
 	CPUs              uint
-	DiskSize          int
+	DiskSize          uint64
```

Comment:
Can you add a comment specifying the disk size unit? I think it's GiB, in which case `uint` is enough.
You can also add a similar comment to `Memory` in the corresponding commit.

Reply:
Done